src/components/record/stones/SkillStoneGem.tsx

```
@@ -13,43 +10,22 @@ interface SkillStoneGemProps {
-  },
-  {
-    id: 5,
-    label: "성찰/성장 원석",
-    Stone: ReflectionGrowthStone,
-    glowClassName: "bg-tag-500",
-    positionClassName: "bottom-[19%] left-1/2 z-40 -translate-x-1/2 rotate-[4deg]",
-  },
-] as const;
+const STONE_ITEMS = (
+  [
+    { id: 1, glowClassName: "bg-tag-200", positionClassName: "top-[22%] left-[31%] z-30 rotate-[-14deg]" },
```

ESLint

[error] Replace `·id:·1,·glowClassName:·"bg-tag-200",·positionClassName:·"top-[22%]·left-[31%]·z-30·rotate-[-14deg]"` with `⏎······id:·1,⏎······glowClassName:·"bg-tag-200",⏎······positionClassName:·"top-[22%]·left-[31%]·z-30·rotate-[-14deg]",⏎···`

(prettier/prettier)